dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -390,6 +391,11 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 									collection,
 									front.isNetworkFront,
 								)}
+								leftContent={
+									collection.displayName === 'Opinion' &&
+									front.config.section === 'commentisfree' &&
+									front.editionId != 'US' && <CPScottHeader />
```

Comment:
I see don't see the Scott header when on the UK edition looking at the `us/commentisfree` front, so I think this should be checking if the pageID is `us/commentisfree` instead? That being said, its a bit odd that we specifically don't show it on the US front, do you know if thats by intetion on Frontend?